src/structures/GuildMember.js

```diff
@@ -284,6 +304,18 @@ class GuildMember extends Base {
     return this.edit({ nick }, reason);
   }
 
+  
+  /**
+   * Disables or enables communication for this member.
+   * @param {?number | Date} seconds The duration in seconds or a date in the future until the member should have
+   * their communication disabled. Passing `null` will enable communication for them. 
+   * @param {string} [reason] Reason for setting the nickname
+   * @returns {Promise<GuildMember>}
+   */
+  disableCommunication(seconds, reason) {
```

Comment:
The name of the first argument doesn't match the one in the typings
Not to mention that `seconds` sounds misleading since it accepts dates